common-utils/src/common/deployment/__init__.py

```diff
@@ -255,17 +226,20 @@ def process(self, build_only: bool = False) -> None:
             build_only (bool, optional): Swtich to pass to methods. Defaults to False.
         """
         self._push_environments(build_only)
-        self._create_file_systems(build_only)
 
 
-class FlowSecret(BaseModel):
-    """Model used to define an ECS Task envar that sources from Secrets Manager.
-    The secret_name must be only the name and not the full ARN.
+class FlowEnvar(BaseModel):
+    """Model used to define an ECS Task envar that
+    either sources from Secrets Manager or is plain text.
+    If a secret, the value must be only the secret name and not the full ARN.
     The deployment process will create the ARN for you.
+
+    Plain text would be a create use case for "EXTRA_PIP_PACKAGES"
+    described here: https://docs.prefect.io/latest/concepts/infrastructure/#installing-extra-dependencies-at-runtime.
     """
 
     envar_name: StrictStr
-    secret_name: StrictStr
+    envar_value: StrictStr
```

Comment:
making FlowSecret into FlowEnvar, which makes it usable for envrionment variable overrides in `FlowDeployment` models